pkg/contracts/annotation.go

```diff
@@ -52,6 +52,12 @@ func getTagValue(layer LayerType) string {
 	switch layer {
 	case Application:
 		return os.Getenv(TagEnvKey)
+	case Os:
+		hostname, _ := os.Hostname()
```

Comment:
@michaelehab  This should return the name of the OS, not the host. We only need to support Linux for now. On Ubuntu, see the `/etc/os_release` file. I would suggest pulling the `PRETTY_NAME` value. In a corresponding annotator to check the OS flavor, this value would be matched against an entry in the SDK config indicating the required OS.